tests/Dfc.CourseDirectory.Core.Tests/Dfc.CourseDirectory.Core.Tests.csproj

```diff
@@ -25,4 +25,10 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="ReferenceDataTests\UkrlpResponse.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
CopyAlways?

Reply:
`PreserveNewest` is the default for things like assets, and seems reasonable to me for this case. I would only flip to always if I knew it was going to cause a problem. Is there a reason for suggesting the change?

Reply:
You're always guaranteed to have the right file with CopyAlways. Say for example you run 0.2 (with a change to this file) on your build pipeline, and then attempt to run 0.1 over the top on the same build agent. PreserveNewest will in theory leave the 0.2 unless the workspace is torn down between builds.

Reply:
I see where you're coming from. Is it worth changing here? The flip side is fractionally increasing the build time by forcing a copy every time. Personally I think I'd wait for it to prove to be an issue, but I'll change it if you want, just let me know.

Reply:
I think this _may_ cause intermittent build failures or at the very least odd behaviours locally. I would personally change it as I like predictable behaviour.

Reply:
Done